sound/soc/sof/ipc4-pcm.c

```diff
@@ -893,11 +893,11 @@ static snd_pcm_sframes_t sof_ipc4_pcm_delay(struct snd_soc_component *component,
 	/* Calculate the delay taking into account that both pointer can wrap */
```

Comment:
"This bug is exposed by a 10 microseconds delay in fw." What does this mean? Was this introduced deliberately to expose the bug?

Reply:
Please check https://github.com/thesofproject/sof/pull/8756. It is a regression issue. The delay is not correct but the 8756 make it the delay to a negative value.

Reply:
you say 10us but that patch talks about 10ms, what am I missing here?

Reply:
sorry,  made mistake about microsecond , it should be milliseconds